<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedLineOptionsDialog</class>
 <widget class="QDialog" name="advancedLineOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance Line Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SerialPortIcons.qrc">
    <normaloff>:/res/images/serial_port_icon.png</normaloff>:/res/images/serial_port_icon.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttons">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="title">
     <string>Freeze Setting</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="rdbNoFreeze">
        <property name="text">
         <string>No Freeze</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbFreeze">
        <property name="text">
         <string>Freeze</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>PWM Sweep</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbPWMSweep"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>PWM Repeat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbPWMRepeat"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sequence Delay</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbSequenceDelay"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="lblSeqRepeat">
      <property name="text">
       <string>Sequence Repeat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbSequenceRepeat"/>
    </item>
   </layout>
  </widget>
  <zorder>buttons</zorder>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>cmbPWMSweep</zorder>
  <zorder>cmbPWMRepeat</zorder>
  <zorder>cmbSequenceDelay</zorder>
  <zorder>lblSeqRepeat</zorder>
  <zorder>cmbSequenceRepeat</zorder>
  <zorder>lblSeqRepeat</zorder>
 </widget>
 <tabstops>
  <tabstop>rdbNoFreeze</tabstop>
  <tabstop>rdbFreeze</tabstop>
  <tabstop>cmbPWMSweep</tabstop>
  <tabstop>cmbPWMRepeat</tabstop>
  <tabstop>cmbSequenceDelay</tabstop>
  <tabstop>buttons</tabstop>
 </tabstops>
 <resources>
  <include location="SerialPortIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>advancedLineOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>advancedLineOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
